bnd.bnd

```diff
@@ -1,5 +1,4 @@
 ScriptEngine-Name:${project.name}
 ScriptEngine-Version:${project.version}
-Import-Package:     org.apache.sling.scripting.bundle.tracker.*;resolution:=optional, \\
-                    *
+Import-Package: *
```

Comment:
Can't we remove this header altogether? I only added it when we were depending on the `bundletracker` API, which was a conditional import.